packages/fds-components/src/components/buttons/ButtonGroup.md

```diff
@@ -0,0 +1,27 @@
+```js
+import { useState } from 'react';
+
+import ButtonGroup  from './ButtonGroup';
+
+import FeedIcon from 'fds-icons/lib/react/FeedIcon';
+import TableIcon from 'fds-icons/lib/react/TableIcon';
+import MarketMapIcon from 'fds-icons/lib/react/MarketMapIcon';
+import StarEmptyIcon from 'fds-icons/lib/react/StarEmptyIcon';
+
+const custom = () => {
+  alert('all unknown props will pass through')
+}
+
+const buttons = [
+  { value: 'Feed', icon: FeedIcon},
+  { value: 'Table', icon: TableIcon},
+  { value: 'Map', icon: MarketMapIcon},
+  { value: 'Suggested', icon: StarEmptyIcon, onClick: custom },
+];
+
+const [firstButton] = buttons;
+
+const [activeValue, setActiveValue] = useState(firstButton.value);
+
+<ButtonGroup buttons={buttons} activeValue={activeValue} onChange={setActiveValue} />
```

Comment:
When the requirements are simple, I really like this approach of passing a simple config into a prop instead of requiring extra components. While thinking this over, I checked the current implementation and it looks like we also must support a "disabled" state.

`activeValue` sets a single active value, which makes this work, but how would we handle disabled states given more than one button could be disabled?

Reply:
Sorry I didn't catch that requirement earlier. There may be an argument for the `GroupButton` component again as a declarative way of setting active/disabled as opposed to the config object and manager pattern